code/src/pages/dashboard/dashboard-orders/DashboardOrdersPage.scss

```diff
@@ -3,6 +3,11 @@
   align-items: center;
   justify-content: space-between;
 
+  &-filter-icons {
+    display: flex;
+    gap: 25px;
```

Comment:
for gap we are using `spacing` from design-system

Reply:
Fixed!